eth/api_tracer_oe.go

```diff
@@ -155,9 +155,7 @@ func (api *PrivateTraceAPI) Block(ctx context.Context, number rpc.BlockNumber, c
 		if err := json.Unmarshal(result.Result.(json.RawMessage), &tmp); err != nil {
 			return nil, err
 		}
-		for _, item := range tmp {
-			results = append(results, item)
-		}
+		results = append(results, tmp...)
```

Comment:
I haven't run actual perf tests, but I've read somewhere that this might be much slower on bigger lists (which is not the case here).

Reply:
Interesting! I just wrote some quick benchmarks to investigate, included in https://github.com/etclabscore/core-geth/pull/211/commits/69d7e09ae8b6d41889de30d328b48ffdecbe4469.

Here are the results:

```txt
goos: linux
goarch: amd64
pkg: github.com/ethereum/go-ethereum/eth
BenchmarkTraceResultsAppend1
BenchmarkTraceResultsAppend1-12    	    3742	    288702 ns/op
BenchmarkTraceResultsAppend2
BenchmarkTraceResultsAppend2-12    	   20686	     59821 ns/op
PASS
```

Reply:
I love you man. Code says always the truth.